### DISCUSSION TITLE
Introducing Andreas S.

### DISCUSSION THREAD

**Discussion by AndreasS2501** [rufuspollock/awesome-crypto-critique#176] [Introductions]
Hi Life-itself! 

My Name is Andreas S. and I'm from Germany. I live with my family near Frankfurt am Main. I work as a Software Developer here in the region. I have been interested in the Blockchain/web3 for a long time from 2015 - 2018. I mainly just read/watched stuff only online. Eventually I found different initiatives like https://maingold.org/ . I do find the concept of local currencies very interesting and have been following/collaborating with the project since some time. 

In that context  I have been following CirclesUBI project  from the Gnosis/Ethereum ecosystem and Neighbourhoods from the Holochain ecosystem.

I really like your perspective offered here of the triangle between individual change / cultural change / systems change.  

In 2015 Charles Hoskinson published Foundations of a Programmable Society,  Diving into the turmoil of him leaving Ethereum is another story but eventually he wrote/experienced the importance of inner change in 2021 https://hoskinsoncharles.blogspot.com/2021/12/i-cant-believe-its-not-buddhism-my-week.html

In 2019 I found John Vervaeke through his series: Awakening from the meaning crisis. Through the related discord community I experienced the benefits of regular wisdom practices. Eventually I hope to see that many practices become more connected to technology space. Leading to a better relationship with technology in a very broad sense. 

I hope to learn more about people involved in lifeitself like for example the hub in berlin which I maybe would like to visit at some point. 

:)

**Comment by rufuspollock** [comment]
@AndreasS2501 welcome!
